biran.gemspec

```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rails", "~> 5.0", ">= 5.0.7"
   s.add_development_dependency "bundler", "~> 2.1"
   s.add_development_dependency 'rspec', '~> 3.7'
+  s.add_development_dependency 'nokogiri', '~> 1.15'
```

Comment:
I think this was the right way to add this, but I may be able to remove this and just used the `bundle update nokogiri` results in the Gemfile.lock that I had to run anyway. I don't fully understand what this list is for, but since 1.14 at a minimum is needed and we are just using what is out now, seems correct to put this here.

Reply:
this is the actual dependency list used when the gem is installed on other projects... the one in Gemfile.lock is more to be used for testing and mock apps or development.

Reply:
It is a development dependency, it will not be installed in the projects. It will basically work as it is in the Gemfile. Looks good to me.